core/language/en-GB/ControlPanel.multids

```diff
@@ -89,15 +89,16 @@ Saving/DownloadSaver/Hint: These settings apply to the HTML5-compatible download
 Saving/General/Caption: General
 Saving/General/Hint: These settings apply to all the loaded savers
 Saving/Hint: Settings used for saving the entire TiddlyWiki as a single file via a saver module
-Saving/GitHub/Branch: Target branch for saving (defaults to `master`)
-Saving/GitHub/Caption: ~GitHub Saver
-Saving/GitHub/Description: These settings are only used when saving to ~GitHub
-Saving/GitHub/ServerURL: Server URL (defaults to `https://api.github.com`)
-Saving/GitHub/Filename: Filename of target file (e.g. `index.html`)
-Saving/GitHub/Password: Password, OAUTH token, or personal access token
-Saving/GitHub/Path: Path to target file (e.g. `/wiki/`)
-Saving/GitHub/Repo: Target repository (e.g. `Jermolene/TiddlyWiki5`)
-Saving/GitHub/UserName: Username
+Saving/GitService/Branch: Target branch for saving (defaults to `master`)
```

Comment:
What does `multids` mean? Does it mean a quick way to create multiple tiddlers?

And how to push text into this file by a plugin, if I want to register some fields into control panel?
I want to create a plugin that can read some custom config from control panel.

Reply:
language multids are for translation text only. you cant store values there